setup.py

```diff
@@ -17,7 +17,7 @@
     install_requires=[
         'djangorestframework>=3.8',
         'requests>=2',
-        'PyJWT==1.7.1',
+        'PyJWT>=1.7.1',
```

Comment:
I think perhaps it's better to pin this to ~=2 So it will always upgrade to a version above 2?